android/src/main/java/com/swmansion/reanimated/layoutReanimation/Snapshot.java

```diff
@@ -115,6 +127,32 @@ public Snapshot(View view) {
     }
     originXByParent = view.getLeft();
     originYByParent = view.getTop();
+    borderRadius = getBorderRadius(view);
+  }
+
+  private float getBorderRadius(View view) {
```

Comment:
feels like this could be a static method extracted to some utils class. It has a clear coupling with RN internals so extracting it out may allow us to provide separate implementations in the future if the internals changes